apps/src/lab2/levelEditors/levelData/EditLibrarySounds.tsx

```diff
@@ -0,0 +1,157 @@
+import React, {useCallback} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import Checkbox from '@cdo/apps/componentLibrary/checkbox/Checkbox';
```

Comment:
super nit: can be simplified to `import Checkbox from '@cdo/apps/componentLibrary/checkbox';`

Reply:
Good catch! Will merge as-is since I have a passing drone build but can clean this up as we add more to this editor